rest/route/host_agent.go

```diff
@@ -1238,22 +1238,14 @@ func (h *hostAgentEndTask) Run(ctx context.Context) gimlet.Responder {
 		endTaskResp.ShouldExit = true
 	}
 
-	// we should disable hosts and prevent them from performing
-	// more work if they appear to be in a bad state
-	// (e.g. encountered 5 consecutive system failures)
+	// Disable hosts and prevent them from performing more work if they have
+	// system failed many tasks in a row.
 	if event.AllRecentHostEventsMatchStatus(ctx, currentHost.Id, consecutiveSystemFailureThreshold, evergreen.TaskSystemFailed) {
 		msg := "host encountered consecutive system failures"
```

Comment:
nit: can we format this string with `consecutiveSystemFailureThreshold`

Reply:
Done.